pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "aws-operator"
 	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "9.1.1"
+	version            = "9.1.1-jakub"
```

Comment:
This should actually have the `-dev` prefix. @giantswarm/team-firecracker-engineers any ideas where this got lost recently? I thought the release workflows take care of that.

Reply:
I'll fix it manually for this PR. Actions have created a PR here: https://github.com/giantswarm/aws-operator/pull/2757, but it was never merged.